docs/index.rst

```diff
@@ -1,9 +1,16 @@
-Hypothesis
-==========
+Welcome to Hypothesis Web Service’s documentation!
```

Comment:
This is a bit pedantic, I realise, but you refer here to “Hypothesis Web Service”, rather than “the Hypothesis web service” below (with the added “the” and the lack of capitals). I’d recommend standardising on one: my personal preference would be for the latter (for consistency with “the Hypothesis client” elsewhere, for one thing).